mungegithub/submit-queue/deployment/kubernetes/configmap.yaml

```diff
@@ -24,8 +24,7 @@ data:
     \"pull-kubernetes-kubemark-e2e-gce\",\
     \"pull-kubernetes-e2e-gce-etcd3\",\
     \"pull-kubernetes-bazel\",\
-    \"pull-kubernetes-e2e-kops-aws\",\
-    \"pull-kubernetes-federation-e2e-gce\""
```

Comment:
Yes, this change tells the submit queue to ignore the federation jobs status on a PR (which may or may not be present).